src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentDiagnosticFactory.cs

```diff
@@ -50,16 +50,6 @@ public static RazorDiagnostic Create_UnexpectedClosingTag(SourceSpan? span, stri
             return RazorDiagnostic.Create(UnexpectedClosingTag, span ?? SourceSpan.Undefined, tagName);
         }
 
-        public static readonly RazorDiagnosticDescriptor MismatchedClosingTag = new RazorDiagnosticDescriptor(
-            "BL9982",
-            () => "Mismatching closing tag. Found '{0}' but expected '{1}'.",
```

Comment:
Decided to not have this error because this means adding more complex logic just for this one case. Mismatched tags will still error though. One for the start tag and one for the end tag.

Reply:
OK. I'm going to look at some examples then. I remember I was especially thorough when I added these. There might be some skipped tests somewhere you can enable

Reply:
I already enabled all three skipped tests.